desktop/apps/consignments2/client/index.js

```diff
@@ -1,13 +1,13 @@
-import ArtworkRailView from '../../../components/artwork_rail/client/view.coffee'
-import Artworks from '../../../collections/artworks.coffee'
-import { data as sd } from 'sharify'
-
 export default () => {
-  // 'Recently Sold' rail
-  const artworks = new Artworks(sd.RECENTLY_SOLD)
-  const rail = new ArtworkRailView({
-    $el: $('.js-recently-sold-rail'),
-    collection: artworks
+  $('body').on('click', '#consignments2-open-faq-modal', (e) => {
```

Comment:
👍  works for me. Since you're using React here though, it might be worth looking into making this a React component—modal's in React are as simple as toggling a boolean state.